tests/IsbnTest.php

```diff
@@ -117,4 +117,14 @@ public function testDoesNotExtractInvalidIsbnAsFromDois()
     {
         $this->assertEmpty(Isbn::extract('http://dx.doi.org/10.978.8898392/316'));
     }
+
+    public function testDoesNotExtractIsbn10sFromHyphenatedIsbn13s()
+    {
+        $this->assertEquals(['9780309570794'], Isbn::extract('978-0-309-57079-4'));
+    }
+
+    public function testDoesNotExtractIsbn10sFromSpaceSeparatedIsbn13s()
+    {
+        $this->assertEquals(['9780309570794'], Isbn::extract('978 0 309 57079 4'));
+    }
```

Comment:
Could you add a test that proves that we don't explicitly remove duplicated ISBNs found? `array_merge` vs `+`